src/types/labEquipment.ts

```
@@ -0,0 +1,40 @@
+export type EquipmentCategory = 'hpc_gpu' | '3d_printer' | 'vr_suite' | 'microscope' | 'laser_cutter';
```

ESLint

[error] Replace `·'hpc_gpu'·|·'3d_printer'·|·'vr_suite'·|·'microscope'·|·'laser_cutter'` with `⏎··"hpc_gpu"·|·"3d_printer"·|·"vr_suite"·|·"microscope"·|·"laser_cutter"`

(prettier/prettier)